### PR TITLE
Add documentation for javascript in the browser

### PR DESCRIPTION
**Pull request by svrnm**
To extend the documentation around javascript I added an additional section for getting started with the browser.

### CODE REVIEW

File: content/en/docs/js/_index.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ weight: 1
 ---
 
 This page contains an introduction to OpenTelemetry in JavaScript. This guide will walk you through installation and instrumentation and show you how to export data.
+=======
+This page contains an introduction to OpenTelemetry in JavaScript. This guide will walk you through installation and instrumentation and show you how to export data.
+>>>>>>> b26cf3c21f64a32d97310d097965e54147f24cb6
```
Comment (austinlparker) [inline content/en/docs/js/_index.md:9]: Looks like there's a bad merge here?

File: content/en/docs/js/_index.md [hunk 2]
```diff
@@ -18,6 +21,9 @@ You can find release information [here](https://github.com/open-telemetry/opente
 # Further Reading
 
 * [OpenTelemetry for JavaScript on GitHub](https://github.com/open-telemetry/opentelemetry-js)
+=======
+* [OpenTelemetry for JavaScript on GitHub](https://github.com/open-telemetry/opentelemetry-js)
+>>>>>>> b26cf3c21f64a32d97310d097965e54147f24cb6
```
Comment (austinlparker) [inline content/en/docs/js/_index.md:26]: Also here

File: content/en/docs/js/_index.md (hunk 1 above)
Comment (austinlparker) [inline content/en/docs/js/_index.md:9]: Bad merge? This should be cleaned up

**Review by austinlparker — CHANGES_REQUESTED** [review]
Maybe instead of having 'getting_started_in_nodejs' and 'getting_started_in_browser', you could move both of them under a new 'getting started' directory under the javascript section? The way it is right now, there's a duplicate 'getting started' page because you didn't remove getting_started.md.

File: content/en/docs/js/_index.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/js/_index.md:9]: Ugh... yes, let me see if I can fix that or if I just do a fresh start.

**Comment by svrnm** [comment]
> Maybe instead of having 'getting_started_in_nodejs' and 'getting_started_in_browser', you could move both of them under a new 'getting started' directory under the javascript section? The way it is right now, there's a duplicate 'getting started' page because you didn't remove getting_started.md.

Good point, let me try that.

File: content/en/docs/js/_index.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/js/_index.md]: Remove that one, thanks

**Review by austinlparker — APPROVED** [review]
looks good!

**Pull request merged**